src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
+    if (isEmpty(data)) throw new HttpException(400, "Groups don't exist");
+    
+    const groups = await this.users.update({
+      where: { id: userId },
+      data
+    })
+
+    return groups;
+  }
```

Comment:
The manageGroups service is not very clear everything points out that it modifies a user, but this is nothing different from a simple patch type edit, this function should be rethought and better delimit its function, documenting with js documentation.

This endpoint should only update the groups within a user and not the whole schema, therefore the data of type Any should be Group[].